<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertPicture</class>
 <widget class="QDialog" name="InsertPicture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>444</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration de l'image</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="filename">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="changePicture">
         <property name="text">
          <string>Changer d'image...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>161</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Affichage de l'image :</string>
         </property>
         <widget class="QRadioButton" name="printAll">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Afficher et imprimer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="noPrint">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Afficher seulement</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="printOnly">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Imp&amp;rimer seulement</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="centerImg">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Centrer l'image</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>158</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>161</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Taille de l'image :</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Largeur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Hauteur :</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="largeur">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>pixels</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>120</y>
            <width>61</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>pixels</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="hauteur">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>91</width>
            <height>28</height>
           </rect>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
         <widget class="QPushButton" name="keepRatio">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>90</y>
            <width>30</width>
            <height>28</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Conserver le ratio</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Ressources.qrc">
            <normaloff>:/images/media_aspect_ratio.png</normaloff>:/images/media_aspect_ratio.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
